src/scenes/home/resetPassword/setPassword/setPassword.js

```diff
@@ -34,19 +34,19 @@ class RequestToken extends Component {
   handleOnClick = (e) => {
     e.preventDefault();
     if (this.isFormValid()) {
-      axios.post(`${config.apiUrl}/users/password`, {
-        user: {
+      axios
+        .post(`${config.apiUrl}/users/passwords/reset`, {
           reset_password_token: this.props.resetPasswordToken,
           password: this.state.password
-        }
-      }).then(() => {
-        this.setState({ success: true, error: null });
-      }).catch(() => {
-        this.setState({ error: 'We were unable to set the password for this email' });
-      });
+        })
+        .then(() => {
+          this.setState({ success: true, error: null });
+        })
+        .catch(() => {
+          this.setState({ error: 'We were unable to set the password for this email' });
```

Comment:
Maybe we should add next steps in this message, such as: "please email staff@operationcode.org for help"

Reply:
I think the toasts are fairly small, and this is a big amount of text, what would you recommend?

Reply:
Just the next steps then. It will be red indicating a failure. It's not accessible, but in terms of getting a message across, let's just say to email us.

Also, is the toaster fixed height? I would hope that would expand to fit content.